output/v3.0/yaml/flavorite/apis/RestaurantsApi.ts

```diff
@@ -0,0 +1,125 @@
+/* tslint:disable */
```

Comment:
add a file .eslintignore in root and specify this folder to ignore whole generated files